iso690-numeric-brackets-cs.csl

```diff
@@ -420,8 +420,15 @@
       <key variable="citation-number"/>
     </sort>
     <layout prefix="[" suffix="]" delimiter=", ">
-      <text variable="citation-number"/>
-      <text variable="locator" prefix=" s. "/>
+      <group delimiter=", ">
+        <group delimiter=" ">
+          <text variable="citation-number"/>
+        </group>
+        <group delimiter=" ">
+          <label variable="locator" form="short"/>
+          <text variable="locator"/>
+        </group>
+      </group>
     </layout>
```

Comment:
Hi zuphilip, your simpification is not working! 
correct citiation: [1 kap. 5, 2, 3 obr. 8]
citation from your simpification: [1kap. 5, 2, 3obr. 8]

correct simplification is:

    <layout prefix="[" suffix="]" delimiter=", ">
          <text variable="citation-number"/>
        <group delimiter=" " prefix=" ">
          <label variable="locator" form="short"/>
          <text variable="locator"/>
        </group>
    </layout>

But I don't like mixing two types of separators (delimiter and prefix) in one tag. I thing that my code is clearer and more understandable.